backend/app/api/endpoints/knowledge.py

```
@@ -767,3 +810,202 @@ def get_qa_history(
+    # Create new session for background task
+    new_db = SessionLocal()
+    try:
+        summary_service = get_summary_service(new_db)
+        await summary_service.refresh_kb_summary(kb_id, user_id)
+    except Exception as e:
+        logger.error(f"Failed to refresh KB summary: {e}")
+    finally:
+        new_db.close()
+
+
+async def _run_document_summary_refresh(doc_id: int, user_id: int, db: Session):
```

Ruff (0.14.10)

Unused function argument: `db`

(ARG001)